src/main/java/core/basesyntax/type/Swimming.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.type;
+
+public interface Swimming {
```

Comment:
can we move `flying` in the separate interface?

Reply:
done